src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -50,7 +50,19 @@ public function getProperties($class, array $context = array())
             return;
         }
 
-        return array_merge($metadata->getFieldNames(), $metadata->getAssociationNames());
+        $properties = array_merge($metadata->getFieldNames(), $metadata->getAssociationNames());
+
+        if (class_exists('Doctrine\ORM\Mapping\Embedded')) {
```

Comment:
As we'll merge this PR in 3.4, you can import this class and do `class_exists(Embedded::class)`.